File: Easy/Java/2220. Minimum Bit Flips to Convert Number.java
class Solution {
    public int minBitFlips(int start, int goal) {
        int xor = start ^ goal; // This will give the xor of start and goal
        int count = 0;
        
        while(xor != 0){
            if((xor & 1) == 1)
                count++;
            xor >>= 1; // Shift right
        }

        return count;
    }
}

File: Easy/Python/2220. Minimum Bit Flips to Convert Number.py
class Solution(object):
    def minBitFlips(self, start, goal):
        xor = start ^ goal # This will give us the xor of start and goal
        count = 0

        while xor != 0:
            if (xor & 1) == 1:
                count += 1
            xor >>= 1 # Shift right
        
        return count
        
